update_gh_pages.sh

```diff
@@ -1,10 +1,37 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
What about movin this script with the other travis tools?

Reply:
That sounds good. In fact, we only need to pass the doc output dir to it.

Reply:
How about merging this to see if it works, and if it does, I'll factor out the duplication into a script in xapi-travis-scripts ?

Reply:
Sounds good to me